v21.2/foreign-key.md

```diff
@@ -26,7 +26,6 @@ To read more about how foreign keys work, see our [What is a Foreign Key? (With
 **Foreign Key Columns**
 
 - Foreign key columns must use their referenced column's [type](data-types.html).
-- A foreign key column cannot be a [computed column](computed-columns.html).
```

Comment:
ditto: see my previous comment for the same line in 21.1 docs

Reply:
Done.